### PR TITLE
RPG Loot Bug on various storages

### LINKED ISSUE #3882: Wizard Item Renaming Spell Bugs
## Round ID:
Any

## Testmerges:
Any

## Reproduction:
Wizard's spell that renames items around the station also renames components, one by one, thus they are no longer stackable. This results in RPEDs with a gargantuan list of individual components.
While funny, it's probably also not great for the server to rename stacking items like this, and a big surprise for anyone that alt-clicks an rped to view contents

### PR DESCRIPTION
**Pull request by Lautarourtiaga**
## About The Pull Request

Fixes #3882 

Basically when a RPGLoot event is triggered. All the items in the game get renamed (as they should) with ridicoulous prefixes and suffixes. 
The thing is that some storages have stackable items, and right now, Each and every one of those stacked items get renamed individually, so they don't stack 😮 (check the screenshots).

**TL;DR**
When an RPGLoot happens certain storages get filled with lot's of the same items.

### Bug Showcase 

The following screenshot has the player with an RPED with some stock parts in it.

part_replacer_items_before_rpgloot_as_it_is_now

The following is the state of the RPED after an RPGLoot event! (those are a lot of repeated items with slighty different names)
 
part_replacer_items_after_rpgLoot_as_it_is_now

### Behavior After Fix

Player with RPED with some stock parts in it.

part_replacer_items_before_rpgLoot_fixed

Now after the RPGLoot event, the items (even though they have different names) stack onto each other without problem.

part_replacer_items_after_rpgloot_fixed 

## Why It's Good For The Game

Because it solves a bug that could be annoying when an RPGLoot happens.

## Changelog
:cl:
fix: Stops RPGLoot from unstacking stackable items
/:cl:

### CODE REVIEW

**Review by jlsnow301 — APPROVED** [review]
great work. the CL might be a bit confusing for someone who hasn't read the pr, though

**Comment by Lautarourtiaga** [comment]
Ow i thought that description was a bit more tecnical!,
fixed

**Comment by DatBoiTim** [comment]
> Ow i thought that description was a bit more tecnical!,
> fixed
CL Change:
fix: Stops RPGLoot from unstacking stackable items

**Comment by Lautarourtiaga** [comment]
There 👍🏻

**Review by PowerfulBacon — APPROVED** [review]
